portfolio/src/main/java/com/google/sps/servlets/CommentsHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class CommentsHandler {
+    private DatastoreService datastore;
+
+    public CommentsHandler() {
+        datastore = DatastoreServiceFactory.getDatastoreService();
+    };
+
+    public String saveComment(Comment comment) {
+        Entity commentEntity = new Entity("Comment");
+        for (Field field: Comment.class.getDeclaredFields()) {
+            try {
+                if (field.get(comment) != null) {
```

Comment:
Is there a reason, why we want this If? Since we are catching the exception in try accept anyway? Also consider adding a variable to avoid doing field.get twice.

Reply:
Yes, I think I need this if. For example, the field `key` is not sent by the client, it is set only after creation. If I set the `key` to null, it will be assigned as a field of datastore object with value null, which I dont want.